libs/ui-watt/src/lib/components/input/timepicker/watt-timepicker.component.scss

```diff
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 @use "@energinet-datahub/watt/utils" as watt;
+@use "../variables" as *;
 
 $placeholder-color: var(--watt-color-neutral-grey-500);
 $input-height-normal: 44px;
```

Comment:
question: do we still need these variables now that we also import them from `variables.scss`?

Reply:
No, we don't. 

I forgot to remove them 🤦‍♂️